caffe2/contrib/fakelowp/layernorm_fp16_fake_op.cc

```diff
@@ -1,8 +1,8 @@
-#include "layernorm_fp16_fake_op.h"
+﻿#include "layernorm_fp16_fake_op.h"
```

Comment:
what is the change here? something with your editor?

Reply:
I think so. I will look into it.